src/frontend/src/eth/utils/erc20.utils.ts

```diff
@@ -64,8 +63,6 @@ const mapErc20Icon = (symbol: string): string | undefined => {
 			return rndr;
 		case 'uni':
 			return uniswap;
-		case 'usdt':
```

Comment:
redo?

Reply:
we don't use that one for USDT anymore, correct? now the icon is included in the `env` file

Reply:
Ah yeah! thanks